server/src/graphql/mutations/createDatabase.ts

```diff
@@ -28,6 +28,18 @@ export const createDatabase: MutationResolvers['createDatabase'] = async (
   // We make sure the name is valid to avoid security risks
   createDatabaseSchema.validateSync({ name: input.name });
 
+  const databases = await prisma.database.findMany({
+    where: {
+      name: input.name,
```

Comment:
@Akirtovskis you can also add the type in the where condition that way you don't need the find afterwards :)

Reply:
@pradel Done! 🙌